packages/account/src/test-utils/launchNode.ts

```diff
@@ -233,7 +233,6 @@ export const launchNode = async ({
       }
       childState.isDead = true;
 
-      removeSideffects();
```

Comment:
Removing this line caused the issue to go away. I suspect that the root cause of the problem is somewhere in the `fuels` package because this line didn't cause problems to e.g. `launchTestNode`. It might be in the interplay between file watchers registered in the `dev` command and this `cleanup` function removing the files they're watching, but I couldn't pinpoint the *actual* root. The cleanup still happens because the `exit` and `error` event listeners registered on the child [above](https://github.com/FuelLabs/fuels-ts/pull/3038/files#diff-b8f07cac51692505e88e065262fd373293a373ed5f6196eb13ec811e37aee136L223) cause the cleanup to happen.